packages/admin/src/Charcoal/Admin/Property/Display/StatusDisplay.php

```diff
@@ -70,7 +70,7 @@ protected function setDependencies(Container $container)
         parent::setDependencies($container);
 
         // Fulfills the ViewableTrait dependencies
-        $this->setView($container['view']);
+        $this->setView($container['admin/view']);
```

Comment:
Since the Admin's `AbstractProperty` requires `admin/view`, `StatusDisplay` does not need to require `admin/view`.

I suggest removing this method here and checking if any other Admin properties are also independently requiring `view`.